Form/DataTransformer/ObjectToIdTransformer.php

```diff
@@ -31,58 +31,87 @@ class ObjectToIdTransformer implements DataTransformerInterface
     protected $repository;
 
     /**
+     * @var boolean
+     */
+    protected $multiple;
+
+    /**
      * @param ManagerRegistry $registry
      * @param string $class
      * @param string $property
      */
-    public function __construct(ManagerRegistry $registry, $om, $class, $property)
-    {
+    public function __construct(ManagerRegistry $registry, $om, $class, $property, $multiple = false) {
```

Comment:
According to [4.3. Methods (PSR-2)](http://www.php-fig.org/psr/psr-2/#methods)
> The opening brace MUST go on its own line,

please fix this
(note that there are a couple of places in code that should be fixed)